virt-v2v/warm/WORKSPACE

```diff
@@ -143,17 +143,16 @@ container_repositories()
 
 container_pull(
     name = "ubi8-minimal",
-    # 'tag' is also supported, but digest is encouraged for reproducibility.
-    digest = "sha256:d1f8eff6032334a81d7cbfd73dacee680e8138db57ecbc91548b97bb45e698e5",
     registry = "registry.access.redhat.com",
     repository = "ubi8/ubi-minimal",
+    tag = "latest",
 )
 
 container_pull(
-    name = "libguestfs-appliance",
-    digest = "sha256:6abc2a68f4a42dbf3a7db1042a73862e763b6c864c1e04659dc6bd981e430ae5",
+    name = "centos-stream-8",
     registry = "quay.io",
-    repository = "kubev2v/libguestfs-appliance",
```

Comment:
here it's internal fork. what did change?

Reply:
it's not a fork - we built the appliance with a dockerfile that is stored in [libguestfs-appliance](https://github.com/kubev2v/libguestfs-appliance), pushed it to quay.io and retrieved it here. with the changes in this PR, it's not needed anymore since we build the appliance like the appliance on el9